translations/base-es.yaml

```diff
@@ -708,21 +707,19 @@ settings:
         compactBuildingInfo:
             title: Información Compacta de Edificios
             description: >-
-                Acorta la caja de información mostrando solo sus ratios. Si no, se mostrara una descripción y una imagen.
+                Acorta la caja de información mostrando solo sus ratios. Si no, se mostrará una descripción y una imagen.
         disableCutDeleteWarnings:
             title: Deshabilitar las advertencias de Cortar/Eliminar
             description: >-
-                Deshabilita los dialogos de advertencia que se muestran cuando se cortan/eliminan mas de 100 elementos.
+                Deshabilita los diálogos de advertencia que se muestran cuando se cortan/eliminan mas de 100 elementos.
 
         enableColorBlindHelper:
-            title: Modo para Daltonicos
-            description: Activa varias herramientas que permiten jugar si eres daltonico.
+            title: Modo para Daltonismo
```

Comment:
Same as in line 63:

"Modo para daltónicos" is the best traduction, "para el daltonismo" feels weird, is not a mode for the condition, but for the people who has the condition.

look in any other software or just look in google for the 2 sentences: "para daltónicos" is the normal sentence in Spanish.